pkg/controller/tidbngmonitoring/tidb_ng_monitoring_control.go

```diff
@@ -46,22 +47,25 @@ type ControlInterface interface {
 }
 
 func NewDefaultTiDBNGMonitoringControl(
-	deps *controller.Dependencies,
+	cli versioned.Interface,
```

Comment:
Why change the deps here?
We may have to add more parameters in deps here later.
If it's for UT, is the code here https://github.com/pingcap/tidb-operator/blob/master/pkg/controller/tidbcluster/tidb_cluster_controller_test.go#L65 useful?

Reply:
just to reduce dependence, use only what it needs.

Reply:
done